Jenkinsfile

```diff
@@ -0,0 +1,36 @@
+stage("test") {
```

Comment:
Not the biggest fan of this approach.  This locks you into Jenkins.  Pulling this back into Gradle or some script you write would be better.